kite-data/kite-data-crunch/src/main/java/org/kitesdk/data/crunch/CrunchDatasets.java

```diff
@@ -96,6 +96,52 @@
   }
 
   /**
+   * Expose the given {@link View} as a Crunch {@link ReadableSource}.
+   *
+   * @param view the view to read from
+   * @param readerSchema the schema used to read from the view
+   * @param <E>     the type of entity produced by the source
+   * @return a {@link ReadableSource} for the view
+   *
+   * @since 1.1.0
+   */
+  public static <E> ReadableSource<E> asSource(View<E> view, Schema readerSchema) {
```

Comment:
I don't think we need anything but `View#asSchema` but I think we should do that instead of a band-aid solution.